### PR TITLE
Removed top NS function call optimization

### PR DESCRIPTION
**Pull request by PEMapModder**

### CODE REVIEW

**Comment by keithkfng** [comment]
Good to see you committing!

**Comment by PEMapModder** [comment]
There is something wrong.

File: src/pocketmine/CrashDump.php [hunk 1]
```diff
@@ -178,17 +178,17 @@ private function baseCrash(){
...
 
-			$reflection = new \ReflectionClass(PluginBase::class);
+			$reflection = new ReflectionClass(PluginBase::class);
 			$file = $reflection->getProperty("file");
 			$file->setAccessible(\true);
 			foreach($this->server->getPluginManager()->getPlugins() as $plugin){
-				$filePath = \pocketmine\cleanPath($file->getValue($plugin));
-				if(\strpos($error["file"], $filePath) === 0){
+				$filePath = \pocketminecleanPath($file->getValue($plugin));
+				if(strpos($error["file"], $filePath) === 0){
```
Comment (PEMapModder) [inline src/pocketmine/CrashDump.php]: this line is wrong.

File: src/pocketmine/CrashDump.php [hunk 2]
```diff
@@ -178,17 +178,17 @@ private function baseCrash(){
...
 		$this->addLine("Type: " . $error["type"]);
 
-		if(\strpos($error["file"], "src/pocketmine/") === \false and \strpos($error["file"], "src/raklib/") === \false and \file_exists($error["fullFile"])){
+		if(strpos($error["file"], "src/pocketmine/") === \false and strpos($error["file"], "src/raklib/") === \false and file_exists($error["fullFile"])){
 			$this->addLine();
 			$this->addLine("THIS CRASH WAS CAUSED BY A PLUGIN");
 			$this->data["plugin"] = \true;
 
-			$reflection = new \ReflectionClass(PluginBase::class);
+			$reflection = new ReflectionClass(PluginBase::class);
```
Comment (PEMapModder) [inline src/pocketmine/CrashDump.php]: this line is also wrong.

**Pull request merged**